project/plugins.sbt

```diff
@@ -1,3 +1,5 @@
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.6")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.6.1")
 addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.12")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
```

Comment:
In current configuration we're not able to see results of coverage, unfortunately. 
My proposition is to use this one: https://coveralls.io (via https://github.com/scoverage/sbt-coveralls)
@manvicha you will need to sync with @mnesvit for this. 
@mnesvit could you create an account for us at coveralls.io?

Reply:
The plugin was added, see this example https://coveralls.io/builds/32961492